cpp/test/core/maybe_tests.cpp

```diff
@@ -64,25 +71,36 @@ TEST_CASE_BEGIN(MaybeInterface)
     UT_AssertIsFalse(z == x);
     UT_AssertIsTrue(z == y);
     UT_AssertIsTrue(x.is_nothing());
+    UT_AssertIsFalse(x);
```

Comment:
Also check for `!x` to make sure that `!` works as expected.

Reply:
Added in the first two places, but not everywhere.